components/search/SearchBarPill.stories.js

```diff
@@ -0,0 +1,14 @@
+import { storiesOf } from '@storybook/vue';
+
+import SearchBarPill from './SearchBarPill.vue';
+
+storiesOf('Search page', module)
```

Comment:
There are no aria labels on the links in Storybook. Why is that?

It would be good to see the default coming through on one of the pills, and a custom aria label on the other.

Each pill should be here as its own "story".